packages/components/grid/src/stories/basics.stories.ts

```diff
@@ -10,7 +10,7 @@ import { type GridColumnDataRenderer } from '../column.js';
 type Story = StoryObj;
 
 export default {
-  title: 'In progress/Grid/Basics'
+  title: 'Layout/Grid/Basics'
```

Comment:
Do we want to set `draft` status for grid as well? :)

Reply:
Grid is a separate folder atm. I'd have to look into that.